app/ts/components/simulationExplaining/customExplainers/SimpleSendVisualisations.tsx

```diff
@@ -82,33 +84,34 @@ function SimpleSend({ transaction, asset, sender, receiver, renameAddressCallBac
 		<span class = 'log-table' style = 'grid-template-columns: min-content min-content min-content; margin-top: 5px;'>
 			<GasFee tx = { transaction } rpcNetwork = { transaction.rpcNetwork } />
 		</span>
+		{ viaProxypath === undefined ? <></> : <span style = 'display: flex;'>
+			<p class = 'paragraph' style = { 'color: var(--subtitle-text-color)' }> Via proxy:&nbsp;</p>
+			<> { interleave(viaProxypath.map((addressBookEntry) => <SmallAddress addressBookEntry = { addressBookEntry } renameAddressCallBack = { renameAddressCallBack }/>), <p class = 'paragraph' style = { 'color: var(--subtitle-text-color)' }>&nbsp;{ '-> '}&nbsp;</p>) } </>
+		</span> }
```

Comment:
semantically, `span` cannot contain block-level element and *may* introduce rendering quirks

```suggestion
		{ viaProxypath === undefined ? <></> : <div style = 'display: flex;'>
			<p class = 'paragraph' style = { 'color: var(--subtitle-text-color)' }> Via proxy:&nbsp;</p>
			<> { interleave(viaProxypath.map((addressBookEntry) => <SmallAddress addressBookEntry = { addressBookEntry } renameAddressCallBack = { renameAddressCallBack }/>), <p class = 'paragraph' style = { 'color: var(--subtitle-text-color)' }>&nbsp;{ '-> '}&nbsp;</p>) } </>
		</div> }
```